media/src/project/project-map.tsx

```diff
@@ -159,6 +169,41 @@ export const ProjectMap: React.FC = () => {
         setMapboxLayers(mapLayers);
     };
 
+    const updateProject = (
+        title: string, description: string, baseMap: string): void => {
+        authedFetch(`/api/project/${projectPk}/`, 'PUT', JSON.stringify(
+            {title: title, description: description, base_map: baseMap})) // eslint-disable-line @typescript-eslint/camelcase, max-len
+            .then((response) => {
+                if (response.status === 200) {
+                    return response.json();
+                } else {
+                    throw 'Project update failed.';
+                }
+            })
+            .then(() => {
+                setProjectTitle(title);
+                setProjectDescription(description);
+                setProjectBaseMap(baseMap);
+            });
+    };
+
+    const deleteProject = (): void => {
+        const csrf = (document.getElementById(
```

Comment:
I am getting a 403 forbidden when attempting to delete a project.

Reply:
Ok, I'll try to replicate this.